driver-core/src/test/unit/com/mongodb/ConnectionStringSpecification.groovy

```diff
@@ -683,6 +683,20 @@ class ConnectionStringSpecification extends Specification {
         connectionString.hosts == ['test5.test.build.10gen.cc']
     }
 
+    // sslEnabled defaults to true with mongodb+srv but can be overridden via query parameter
+    def 'should set sslEnabled property with SRV protocol'() {
+        expect:
+        connectionString.getSslEnabled() == sslEnabled
+
+        where:
+        connectionString                                                           | sslEnabled
+        new ConnectionString('mongodb+srv://test5.test.build.10gen.cc')            | true
+        new ConnectionString('mongodb+srv://test5.test.build.10gen.cc/?tls=true')  | true
+        new ConnectionString('mongodb+srv://test5.test.build.10gen.cc/?ssl=true')  | true
+        new ConnectionString('mongodb+srv://test5.test.build.10gen.cc/?tls=false') | false
```

Comment:
This is the only one that failed.  The others are just to increase test coverage.